charts/flyte-core/values-sandbox.yaml

```diff
@@ -102,3 +117,64 @@ redis:
   # --- enable or disable Redis Statefulset installation
   enabled: false
 
+# -- Configuration for the Cluster resource manager component. This is an optional component, that enables automatic
+# cluster configuration. This is useful to set default quotas, manage namespaces etc that map to a project/domain
+cluster_resource_manager:
```

Comment:
Why do we need this block?

Reply:
IIUC, If we don't add it, the sandbox will use the default config in [value.yaml](https://github.com/flyteorg/flyte/blob/173c409e2633a7fa5a33ad07491665ef35acfe6f/charts/flyte-core/values.yaml#L732)

> How did you test this, Kevin?

I rebuilt the sandbox image and ran the command `flytectl sandbox start --image pingsutw/sandbox-test` to create cluster. Then check if the resource is updated in the configmap.